src/pages/items.jsx

```diff
@@ -0,0 +1,127 @@
+import { useEffect, useRef, useState } from 'react';
+import { Link } from "react-router-dom";
+import fetchLists from '../api/fetchCardList';
+import Card from '../components/card';
+import Button from '../components/button';
+import Input from '../components/input';
+import Avatar from '../public/items/avatar.png';
+import NavBarLogo from '../public/navbar-button.png';
+import SearchIcon from '../public/items/ic_search.png';
+import './items.css';
+import Select from '../components/select';
+import Pagination from '../components/pagination';
+
+const orderSelect = [
+  {
+    name: "최신순",
+    value: 'recent'
+  },
+  {
+    name: "좋아요순",
+    value: 'favorite'
+  }
+]
+
+function Items() {
+  const [bestCardData, setBestCardData] = useState([]);
+  const [cardData, setCardData] = useState([]);
+  const [sort, setSort] = useState('recent');
+  const [currentPage, setCurrentPage] = useState(1);
+  const totalCount = useRef(0);
+  const totalPage = useRef(0);
+
+  const ITEMS_PER_PAGE = 10;
+
+  const handleChange = (el) => {
+    if (el === 'favorite') {
+      setSort('favorite');
+      fetchLists(ITEMS_PER_PAGE, 1, el).then((res) => setCardData(res.list));
+    } else if (el === 'recent') {
+      setSort('recent');
+      fetchLists(ITEMS_PER_PAGE, 1, el).then((res) => setCardData(res.list));
+    }
```

Comment:
el 값에 따라 분기가 되지 않고있는데, 간단하게 이렇게 고쳐보면 어떨까요?

```suggestion
    setSort(el);
    fetchLists(ITEMS_PER_PAGE, 1, el).then((res) => setCardData(res.list));
```

만약 여기서 해당 로직 실행을 걸러줘야하는 조건 분기가 생긴다면, early return을 사용해주시는게 더 깔끔할거예요!

[참고](https://velog.io/@qhflrnfl4324/%EB%82%91%EA%B9%A1%ED%8C%9C07)